frontend/src/features/entity_selector/EntitySelector.tsx

```diff
@@ -296,16 +312,39 @@ const EntitySelectorInnerAuth = ({
           </Box>
         </>
       )}
-
-      {rating ? (
-        <EntityCard
-          compact
-          entity={rating.entity}
-          settings={showRatingControl ? toggleAction : undefined}
-        />
-      ) : (
-        <EmptyEntityCard compact loading={loading} />
-      )}
+      <Box position="relative">
+        {(rating && (
+          <EntityCard
+            compact
+            entity={rating.entity}
+            settings={showRatingControl ? toggleAction : undefined}
+          ></EntityCard>
+        )) || <EmptyEntityCard compact loading={loading} />}
```

Comment:
(nitpicking) 
Is it not equivalent to the previous version, that I find more readable?

```suggestion
        {rating ? (
          <EntityCard
            compact
            entity={rating.entity}
            settings={showRatingControl ? toggleAction : undefined}
          ></EntityCard>
        ) : (
          <EmptyEntityCard compact loading={loading} />
        )}
```

Reply:
Good point!